src/cloud-element-templates/properties-panel/properties/custom-properties/index.js

```diff
@@ -132,7 +132,7 @@ function createCustomEntry(id, element, property) {
     return {
       id,
       component: FeelProperty,
-      isEdited: isFeelEntryEdited,
+      isEdited: isEdited(isFeelEntryEdited,property),
```

Comment:
`isEdited` property is a callback:

```suggestion
      isEdited: () => isEdited(isFeelEntryEdited, property),
```

Reply:
OK I see that you attempted to make it a factory function, so let's better rename it instead:

```suggestion
      isEdited: createIsEdited(isFeelEntryEdited, property),
```